src/SprykerEco/Client/Payone/ClientApi/Request/AbstractRequest.php

```diff
@@ -102,25 +102,25 @@ public function getEncoding(): ?string
      *
      * @return void
      */
-    public function setKey($key)
+    public function setKey(string $key): void
     {
         $this->key = md5($key);
     }
 
     /**
      * @return string
      */
-    public function getKey()
+    public function getKey(): string
     {
         return $this->key;
     }
 
     /**
-     * @param string $mid
+     * @param string|null $mid
```

Comment:
Why did you add null here as well?

Reply:
It's because we have such cases, when we put something which has string|null as a return type hint to this variable.